README.md

```diff
@@ -38,11 +75,12 @@ By default, this package looks for your Salesforce data in the `salesforce` sche
 config-version: 2
 
 vars:
+    salesforce_database: your_database_name    
     salesforce_schema: your_schema_name
-    salesforce_database: your_database_name
+    salesforce__<default_source_table_name>_identifier: your_table_name
```

Comment:
Since you already included this in the optional section below, we should remove it here.